### ISSUE TITLE
Issue with Tinkers

### ISSUE THREAD

**Issue by IppSwitch** [kreezxil/World-of-Dragons#317]
**TL;DR - Incorrect stat values being shown for custom created tinkers tools in the tool forge.** 

While testing various things i've come across a very minor bug with Tinkers where the correct stats of the completed item are not the same as the one displayed WITHIN the Tinkers Tool Forge. 
While creating a Tinkers hammer out of Cobalt (Head), Electrum (Large plate), Knightslime (Large Plate + Tough Tool Rod). (We'll call this the Mixed Hammer)
![Discrepency](https://user-images.githubusercontent.com/38470419/63995408-f5896d00-cb2a-11e9-8c00-af169eea6f16.png)

Also when adding Diamond or Emerald modifier for a custom created tinker tool, the overall stats are lowered and you are given a lesser version of the created item. This is all of the different modifier results on the Mixed Tinkers Hammer.
![New or Old](https://user-images.githubusercontent.com/38470419/63995140-fa99ec80-cb29-11e9-83c9-48f167de35fa.png)

Here is one made from JUST Cobalt. 
![Vanilla cobalt](https://user-images.githubusercontent.com/38470419/63995191-3df45b00-cb2a-11e9-99a3-fef3822f588d.png)

I believe that the base vanilla tool when adding certain or specific modifiers will force the tool to revert to it's base primary stats. (The stats of the main component, in this case the Tool Head)

I have not yet tested anything other than the hammer so far. Or anything other than Cobalt. I will continue to test as i go.

**Comment by hellwolf95** [comment]
Stated the reason in another post

**Comment by IppSwitch** [comment]
Yea i won't be sad to see them go, but me being me, i see a bug and or issue and i still report on it lol.

**Comment by kreezxil** [comment]
This is caused by Tweakers Construct, the only solution is to remove
Tweakers Construct, removing Tweakers Construct, causes players to wonder
why it's been removed and allows Tinkers' Tools to be the GO TO DeFact GOD
LEVEL TOOLS of choice.

I'll be removing the ENTIRE Tinkers' suit in M19 if it should ever become a
reality.

TINKERS will not enter into the new pack which I decided finally is the way
to go and will begin creating in earnest on Sunday.

**Comment by IppSwitch** [comment]
Hooray :D Looking forward to the next part of the pack.

**Comment by kreezxil** [comment]
After i'm done with BANE OF DRAGONS, I'll remove everything but Tinker's from build 224, that will solve this problem once and for all.

**Comment by kreezxil** [comment]
in hind sight this looks like it was caused by tweakers construct not actually being on the server. I still didn't care for tinkers tho as it encouraged players to make that ultimate tinker weapon and armor set isntead of actually playing and in the end allowed them to solo dragons, also not what I wanted. ever.

all tinkers have been removed in m19 due out today, world wipe included.
